__tests__/integration/multisig.test.js

```diff
@@ -300,4 +321,715 @@ describe('send tx (HTR)', () => {
+            },
+            script: expect.any(String),
+            token: '00',
+            tokenData: 0,
+            token_data: 0,
+            type: 'p2sh',
+            value: 1,
+          },
+        ],
+      },
+      balance: {
+        '00': {
+          tokens: { available: 1, locked: 0 },
+          authorities: {
+            melt: { available: 0, locked: 0 },
+            mint: { available: 0, locked: 0 },
+          },
+        },
+        [txCreateToken.hash]: {
+          tokens: { available: -100, locked: 0 },
+          authorities: {
+            melt: { available: 0, locked: 0 },
+            mint: { available: 0, locked: 0 },
+          },
+        },
+      },
+    });
+  });
+
+  // create-token
+  it('should fail to send a create-token transaction with less than minimum signatures', async () => {
+    const address = precalculatedMultisig[0].addresses[0];
+    let response = await TestUtils.request
+      .post('/wallet/p2sh/tx-proposal/create-token')
+      .send({
+        name: 'My Custom Token',
+        symbol: 'MCT',
+        amount: 100,
+        address,
+      })
+      .set({ 'x-wallet-id': wallet1.walletId });
+    expect(response.body.success).toBe(true);
+
+    const { txHex } = response.body;
+
+    // collect signatures from 3 wallets
+    const signatures = await TestUtils.getXSignatures(txHex, wallets, 2);
+
+    // try to send
+    response = await TestUtils.request
+      .post('/wallet/p2sh/tx-proposal/sign-and-push')
+      .send({ txHex, signatures })
+      .set({ 'x-wallet-id': wallet1.walletId });
+
+    await TestUtils.pauseForWsUpdate();
+
+    expect(response.status).toBe(200);
+    expect(response.body.success).toBe(false);
+    expect(response.body.error).toBeDefined();
+  });
+
+  it('Should fail to send a create-token transaction with incorrect signatures', async () => {
+    const address = await wallet1.getAddressAt(0);
+    let response = await TestUtils.request
+      .post('/wallet/p2sh/tx-proposal/create-token')
+      .send({
+        name: 'My Custom Token',
+        symbol: 'MCT',
+        amount: 100,
+        address,
+      })
+      .set({ 'x-wallet-id': wallet1.walletId });
+    expect(response.body.success).toBe(true);
+
+    const { txHex } = response.body;
+
+    // collect signatures from 3 wallets
+    const signatures = await TestUtils.getXSignatures(txHex, wallets, 2);
+    const pubkey = precalculatedMultisig[0].multisigDebugData.pubkeys[0];
+    const invalidP2shSig = new hathorLib.P2SHSignature(pubkey, {});
+    signatures.push(invalidP2shSig.serialize());
+
+    // try to send
+    response = await TestUtils.request
+      .post('/wallet/p2sh/tx-proposal/sign-and-push')
+      .send({ txHex, signatures })
+      .set({ 'x-wallet-id': wallet1.walletId });
+
+    await TestUtils.pauseForWsUpdate();
+
+    expect(response.status).toBe(200);
+    expect(response.body.success).toBe(false);
+    expect(response.body.error).toBeDefined();
```

Comment:
Same here. No need to add the whole message on the validation, but maybe just a relevant substring such as "invalid signatures".

I believe the remaining error handling tests below could also benefit from this improvement.